README.md

```diff
@@ -9,31 +9,38 @@ If you want to know how to use it as replacement for
 [Drush Make](https://github.com/drush-ops/drush/blob/8.x/docs/make.md) visit
 the [Documentation on drupal.org](https://www.drupal.org/node/2471553).
 
-## Usage
+## Quick installation
+
+All of the necessary configuration to build the site is stored in the `config` directory. The easiest way to get it all running is to build it locally using Composer and Drush.
 
 First you need to [install composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx).
 
 > Note: The instructions below refer to the [global composer installation](https://getcomposer.org/doc/00-intro.md#globally).
 You might need to replace `composer` with `php composer.phar` (or similar) 
 for your setup.
 
-After that you can create the project:
+After that all of the composer requirements need to be installed:
 
 ```
-composer create-project drupal-composer/drupal-project:8.x-dev some-dir --stability dev --no-interaction
+composer install
 ```
 
-With `composer require ...` you can download new dependencies to your 
-installation.
+Installing the ContribKanban site locally is easiest using sqlite:
 
 ```
-cd some-dir
-composer require drupal/devel:~1.0
+cd web
+drush site-install minimal --db-url="sqlite://../private/db.sqlite" --account-mail="youremail@example.com" --account-name="admin" --account-pass="test" --site-mail="webmaster@example.com" --site-name="Contrib Kanban"
```

Comment:
I think we can drop `minimal` and `--db-url`

```
$databases['default']['default'] = array (
  'database' => '../private/db.sqlite',
  'prefix' => '',
  'namespace' => 'Drupal\\Core\\Database\\Driver\\sqlite',
  'driver' => 'sqlite',
);
$settings['install_profile'] = 'minimal';
```

That is committed, but haven't tested!

Reply:
According to "drush site-install --help":

> profile
> The install profile you wish to run. Defaults to 'default' in D6, 'standard' in D7+, unless an install profile is marked as exclusive (or as a distribution in D8+ terminology) in which case that is used.

That suggests that if "minimal" isn't provided then it'll use "standard" by default, which then means it won't be able to pull in the config because the exported config says to change to the "minimal" install profile after the site is already installed.

Reply:
I'll test without the --db-url argument..